### PR TITLE
Added brand, workflow and snap sections with placeholder images

### PR DESCRIPTION
**Pull request by bartaz**
screen shot 2017-02-15 at 13 23 43

Also included bottom section about snap (and footer fixes):

screen shot 2017-02-15 at 15 11 57

### CODE REVIEW

File: src/common/containers/container.css [hunk 1]
```diff
@@ -2,11 +2,12 @@
   box-sizing: border-box;
   max-width: 940px;
   margin: 0 auto;
-  padding: 10px;
+  padding: 0 10px;
 }
 
 .container {
   composes: wrapper;
+  padding: 10px;
```
Comment (bartaz) [inline src/common/containers/container.css]: Not sure if container needs top/bottom padding at all, but as other pages use it too I keep it around for backwards compatibility.

File: src/common/components/footer/footer.css [hunk 2]
```diff
@@ -7,7 +7,7 @@ html {
 }
 
 body {
-  margin-bottom: 75px;
+  margin-bottom: 135px;
```
Comment (bartaz) [inline src/common/components/footer/footer.css]: Footer was updated and is now bigger then in MU.

**Comment by nottrobin** [comment]
[Throwaway card](https://trello.com/c/R9lr0o2K/130-throwaway-card)

File: src/common/containers/landing.js [hunk 3]
```diff
@@ -43,31 +43,54 @@ class Landing extends Component {
...
 
-          {/* TODO testimonials */}
-        </div>
+        <section className={styles.section}>
+          <div className={ `${styles.row} ${containerStyles.wrapper}` }>
+
+
+            <div className={styles.rowItemTwoThirds}>
+              <p className={styles.snaps}>Snaps are secure, sandboxed, containerised applications, packaged with their dependencies for predictable behavior. With the Snap Store, people can safely install apps from any vendor on mission-critical devices and PCs.</p>
+              <Anchor href="http://snapcraft.io">More about snaps</Anchor>
```
Comment (cjwatson) [inline src/common/containers/landing.js]: This should be `https` from the start, since the `http` version redirects to `https` anyway.

**Pull request merged**